src/cli/auth.rs

```diff
@@ -0,0 +1,56 @@
+use clap::Parser;
+use rattler_networking::{Authentication, AuthenticationStorage};
+
+// use crate::auth::{default_authentication_storage, listen_and_open_browser};
+
+fn default_authentication_storage() -> AuthenticationStorage {
+    AuthenticationStorage::new("rattler", &dirs::home_dir().unwrap().join(".rattler"))
+}
+
+/// Adds a dependency to the project
+#[derive(Parser, Debug)]
+pub struct Args {
+    /// The host to authenticate with
+    host: String,
+
+    /// The token to use (for bearer authentication)
```

Comment:
Not sure if bearer authentication is going to help the user. Maybe just name it API token for prefix.dev.